requirements.md

```diff
@@ -16,29 +16,46 @@ See also: [SPC Scope](scope.md) for definitions and more information.
 ## Web Context Support
 
 * It must possible to invoke SPC from an iframe. See [issue 68](https://github.com/w3c/secure-payment-confirmation/issues/68).
+
+Q: I understand that woudl also require PR API to be used from within the IFrame. Is that right?
+
 * It must possible to invoke SPC from within Payment Request API.
+
+Q: Shouldn't that be rather "It must be possible to invoke SPC via Payment Request API"? I understand the intention is to use the PR API as a way to invoke the SCP.
+
 * It must possible to invoke SPC from within a Payment Handler.
 
+Q: Does is mean the Payment Handler API woudl need to be extended by some new capabilities to trigger the SPC from within the PH code?
```

Comment:
Possibly, if SPC ends up using the PR API as a basis. One cannot invoke PR from within a PaymentHandler (because 
 PaymentRequest.show() cannot be nested), so some other API would be needed in that case.

Straw-person _example_ of how this could work: it could be possible both to trigger SPC via `new PaymentRequest(...).show()` **and** adding a new method to [PaymentRequestEvent](https://www.w3.org/TR/payment-handler/#the-paymentrequestevent) called `invokeSPC()`.

Reply:
i've added a note to the requirements document that this is likely to require updates to PH API.